call_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+)
+
+func TestClient(t *testing.T) {
+
+	tt := []struct {
+		name    string
+		method  string
+		path    string
+		payload string
+	}{
+		{name: "update", method: "PUT", path: "/rest/api/2/user/rest/api/2/user?username=foo%40bar.com", payload: `{"emailAddress":"foo@bat.com"}`},
```

Comment:
Bad path expected here in the test or a mistake?

Reply:
good catch as always Mr Laser Eyez, fixed